ui/src/message_popup/transformations.jsx

```diff
@@ -8,8 +8,13 @@ import {allQuestions} from './questions.js';
 
 export function withStudents(questions) {
   return questions.map((question) => {
-    const student = _.find(allStudents, {id: question.studentId });
-    return _.extend({student}, question);
+    var students = [];
+    if(_.has(question, 'studentIds')){
```

Comment:
I'd recommend using `.map` to transform the `studentIds` over using a for loop to iterate through the array yourself.